### DISCUSSION TITLE
Change YAML setting via CLI: how to do it?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#884] [Q&A] [answered]
Hi,
I would like to run something like the command below, to apply only in rendering `self-contained:true`.

```
quarto render input.qmd pandoc-args self-contained:true
```

I have tried using `-M` and `-P` options, but these do not work, because `self-contained` is probably neither Metadata nor Parameter.

How to do it?

Thank you

**Comment by jjallaire** [comment, marked as answer]
```
quarto render input.qmd --self-contained
```

**Reply by aborruso** [reply]
@jjallaire as usual thank you very much.

If I run `quarto --help` I do not have it, and it seems to me that it's not in documentation website.

Where can I read something about this and the other similar options?

**Comment by jjallaire** [comment]
We do document that we support all pandoc command line options. You can find documentation for these options here: https://pandoc.org/MANUAL.html#options
